alpine-aws/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# dkubb/alpine-aws
+
+FROM alpine:3.6
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+# Upgrade system dependencies
+RUN apk upgrade --update-cache --available
+
+# Install system dependencies
+RUN apk add --update-cache py2-pip=9.0.1-r1
```

Comment:
Since we perform `--update-cache` on the line above, is it necessary to do it again here?